Numpy is not defined as nifgen_system_tests dependency.

* Description of issue

    nifgen_system_tests uses numpy module for the test,
https://github.com/ni/nimi-python/blob/6c87ee9824addb68a71ee24591fad813266be9c5/src/nifgen/system_tests/test_system_nifgen.py#L96
https://github.com/ni/nimi-python/blob/6c87ee9824addb68a71ee24591fad813266be9c5/src/nifgen/system_tests/test_system_nifgen.py#L102

But tox deos not have listed numpy as dependency for nifgen_system_tests 
https://github.com/ni/nimi-python/blob/6c87ee9824addb68a71ee24591fad813266be9c5/tox.ini#L205

* Error
    ```src\nifgen/system_tests/test_system_nifgen.py:3: in <module>
       import numpy
       E   ImportError: No module named numpy

* Steps to reproduce issue
    1. Run test for nifgen
    2. Observe that it give the error

--- a/tox.ini
+++ b/tox.ini
@@ -70,12 +70,14 @@ changedir =
 commands =
     build_test: python --version
     build_test: python -c "import platform; print(platform.architecture())"
+    build_test: python -m pip install pip --upgrade pip
     build_test: touch .coverage
     build_test: rm .coverage
     build_test: coverage run --rcfile=tools/coverage_unit_tests.rc --append --source build.helper -m py.test --pyargs build.helper
     build_test: flake8 build/
     test: python --version
     test: python -c "import platform; print(platform.architecture())"
+    test: python -m pip install pip --upgrade pip
     test: coverage run --rcfile=tools/coverage_unit_tests.rc --append --source nifake -m py.test bin/nifake/nifake {posargs} -s
     test: coverage run --rcfile=tools/coverage_unit_tests.rc --append --source nimodinst -m py.test bin/nimodinst/nimodinst {posargs} -s
     test: coverage report --rcfile=tools/coverage_unit_tests.rc
@@ -85,9 +87,11 @@ commands =
     clean: make clean {posargs}
     codegen: python --version
     codegen: python -c "import platform; print(platform.architecture())"
+    codegen: python -m pip install pip --upgrade pip
     codegen: make {posargs} PYTHON_CMD=python
     flake8: python --version
     flake8: python -c "import platform; print(platform.architecture())"
+    flake8: python -m pip install pip --upgrade pip
     flake8: flake8 bin/
     flake8: flake8 src/nidcpower/system_tests/
     flake8: flake8 src/nidcpower/examples/

--- a/tox.ini
+++ b/tox.ini
@@ -202,11 +202,13 @@ deps =
     nidmm_system_tests: pytest
     nidmm_system_tests: nidmm
     nidmm_system_tests: pytest-json
+    nidmm_system_tests: numpy
     nifgen_system_tests: enum34;python_version<"3.4"
     nifgen_system_tests: singledispatch;python_version<"3.4"
     nifgen_system_tests: pytest
     nifgen_system_tests: nifgen
     nifgen_system_tests: pytest-json
+    nifgen_system_tests: numpy
     niscope_system_tests: enum34;python_version<"3.4"
     niscope_system_tests: singledispatch;python_version<"3.4"
     niscope_system_tests: pytest

